packages/dds/tree/src/test/feature-libraries/typedSchema/typeUtils.spec.ts

```diff
@@ -1,24 +0,0 @@
-/*!
- * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
- * Licensed under the MIT License.
- */
-
-import {
-	ArrayToUnion,
-	// Allow importing from this specific file which is being tested:
-	/* eslint-disable-next-line import/no-internal-modules */
-} from "../../../feature-libraries/typed-schema/typeUtils.js";
-import { areSafelyAssignable, isAssignableTo, requireTrue } from "../../../util/index.js";
-
-// These tests currently just cover the type checking, so its all compile time.
-
-// Test ArrayToUnion
```

Comment:
FlexListToUnion, the one user of ArrayToUnion, has tests for these cases already, so removing these should not result in a loss of coverage.